docs/adrs/0003-support-aggregations-on-publisher-name.md

```diff
@@ -0,0 +1,98 @@
+| |   | PublicationInformation  | Elsevier |
+| |   | PublicationInformation  | Wiley |
+| ArchivesSpace | XML select array of Strings matching element `dc:publisher`  | PublicationInformation  | Massachusetts Institute of Technology. Libraries. Department of Distinctive Collections (note: this appears to be effectively a static string for all of our ASpace records) |
+| MIT GIS |   | PublicationInformation |  |
+| OGM GIS |   | PublicationInformation  |  |
+
+### Proposed mappings (Option 1)
+
+Create a new object `PublicationInfo` and map data into it. Most sources just use `Publication.name` but Alma uses additional fields.
+
+```json
+{
+  "publicationInfo": [
+    {
+      "name": "String",
+      "location": "String",
+      "date": "String (ideally dates would be dates... but that may be out of scope for now)"
+    }
+  ]
+}
+```
+
+Notes: we should consider whether this really needs to be an array. 260 and 264 are repeatable fields in MARC, but whether they are regularly used in that way or if we could instead pick the "first" one that shows up and get a simpler model without losing any real world functionality would be worth investigating. For now it is modeled as an array of `PublicationInfo` assuming that is appropriate.
+
+GraphQL would collapse the new object and continue to serve it as the deprecated field `PublicationInformation` until we are confident it is no longer being used.
+
+| Source | Mappings | TIMDEX Field | Notes |
+| --- | --- | --- | --- |
+| Alma | [MARC 260$a](https://www.loc.gov/marc/bibliographic/bd260.html) | PublicationInfo.Location | |
+| Alma | [MARC 264$a](https://www.loc.gov/marc/bibliographic/bd264.html) | PublicationInfo.Location | |
+| Alma | [MARC 260$b](https://www.loc.gov/marc/bibliographic/bd260.html) | PublicationInfo.Name | |
+| Alma | [MARC 264$b](https://www.loc.gov/marc/bibliographic/bd264.html) | PublicationInfo.Name | |
+| Alma | [MARC 260$c](https://www.loc.gov/marc/bibliographic/bd260.html) | PublicationInfo.Date | |
+| Alma | [MARC 264$c](https://www.loc.gov/marc/bibliographic/bd264.html) | PublicationInfo.Date | |
+| Alma | [MARC 260$d](https://www.loc.gov/marc/bibliographic/bd260.html) | Invalid field. Don't map | |
+| Alma | [MARC 260$e](https://www.loc.gov/marc/bibliographic/bd260.html) | Don't map | |
+| Alma | [MARC 260$c](https://www.loc.gov/marc/bibliographic/bd260.html) | Don't map | |
+| DSpace | Keep logic | Publication.Name | note: consider future work to normalize in source or during transform |
+| ASpace | Keep logic | PublicationInfo.Name | |
+| GIS MIT | | PublicationInfo.Name | |
+| GIS OGM | | PublicationInfo.Name | |
+
+### Proposed mappings (Option 2)
+
+Move some of the Alma data to other existing objects, and create a new top level `Publisher` array of strings.
+
+All sources except Alma remap to use `Publisher` instead of `PublicationInformation` with no other changes at this time.
```

Comment:
Curious about this line:
> All sources except Alma remap to use `Publisher` instead of `PublicationInformation` with no other changes at this time.

I realized on my first read, I had made the assumption:
- all sources (including Alma) would begin to write the publisher *name* to this new multivalued string `publisher` field
- future work: any sources (including Alma) that have publisher date or location information, could write to other appropriate fields, e.g. `dates` or `locations` with a qualifier like `@kind="Publisher"`

Does this align with your thinking and the wording here?

Reply:
@JPrevost Does this change mean `TimdexRecord.publication_information`/`PublicationInformation` will be deprecated? 🤔

Reply:
@JPrevost - language updated in Option 2, per [your aggrement with a slight rewording](https://github.com/MITLibraries/transmogrifier/pull/132#discussion_r1506082129), per this commit: https://github.com/MITLibraries/transmogrifier/pull/132/commits/1dfcfab19a41d5a2c021658e7acd9186c79afb53.